seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -203,11 +203,8 @@ def test_user_get_workspace_access_level(self, mock_time, mock_redis, mock_logge
 
     @responses.activate
     @mock.patch('seqr.views.utils.terra_api_utils.SERVICE_ACCOUNT_FOR_ANVIL', TEST_SERVICE_ACCOUNT)
-    @mock.patch('seqr.utils.redis_utils.redis.StrictRedis')
     @mock.patch('seqr.views.utils.terra_api_utils.time')
-    def test_add_service_account(self, mock_time, mock_redis):
-        mock_redis.return_value.get.return_value = None
-
+    def test_add_service_account(self, mock_time):
```

Comment:
The mock of the `redis` is never used in this test. I didn't take them out in last PR.